manual/src/ornate/reference.md

```diff
@@ -68,6 +68,17 @@ useYarn := true
 
 The `yarn` command must be available in the host platform.
 
+### Reload Workflow {#reload-workflow}
+
+To get a faster "change source and reload application" workflow enable the setting `enableReloadWorkflow`.
```

Comment:
And I would also start with the following paragraph:

> Each time you change something in your application source code and compile the project, Scala.js emits a new .js file that can weigh several MBs (in `fastOptJS` mode) if your application is large. Turning this huge CommonJS module into code executable by web browsers takes time. This can be a problem if you rely on a “live-reloading” workflow (like the one Play framework, for instance), because the reloading time can go up to 30 seconds.